data-service-client/src/main/java/com/amazonaws/blox/dataservice/client/v1/model/CreateEnvironmentResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.amazonaws.blox.dataservice.client.v1.model;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@Builder
+public class CreateEnvironmentResponse {
+
+    @NonNull
+    private String environmentVersion;
```

Comment:
It looks like most of the create APIs on ECS returns the entire created resource, rather than just an identifier for it (e.g. [createCluster](http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateCluster.html#API_CreateCluster_ResponseSyntax)). Should we be following the same kind of pattern here?